### PR TITLE
Update env-vars

### PR DESCRIPTION
**Pull request by alistar79**
var miss match between env-vars and variables.tf

### CODE REVIEW

**Comment by rohanbhattacharjee** [comment]
We have some inconsistencies in the examples in how certain variables are read and handled.
Like we found with the ssh key, some examples expect the content of the key, some others expect the key path is passed in.
compartment_id vs. compartment_ocid is another one that is inconsistent.

I have logged in internal bug to decide on what we prefer, and make all the examples consistent with it (instead of doing it one example at a time).

File: docs/examples/compute/instance/env-vars [hunk 1]
```diff
@@ -11,7 +11,7 @@ export TF_VAR_region="<region in which to operate, example: us-ashburn-1, us-pho
 export TF_VAR_compartment_ocid="<compartment OCID>"
 
 ### Public/private keys used on the instance
-export TF_VAR_ssh_public_key_path=<path to public key>
-export TF_VAR_ssh_private_key_path=<path to private key>
+export TF_VAR_ssh_public_key=<path to public key>
```
Comment (rcohenma) [inline docs/examples/compute/instance/env-vars]: "<path to public key>" should also be changed to "<public key>"
same for the private key

Comment (rcohenma) [reply docs/examples/compute/instance/env-vars]: It seems we have a change in progress that will take care of this. It will be out soon.

**Comment by briangustafson** [comment]
Thanks again for pointing out this issue. I see that this has since been addressed by a separate change at https://github.com/oracle/terraform-provider-oci/commit/c27090090dfab0a6e5d28eb8974b1809337d718f, so closing this PR.

**Pull request closed without merging**